adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -686,19 +692,19 @@ def red_led(self):
         return self._led.value
 
     @red_led.setter
-    def red_led(self, value):
+    def red_led(self, value) -> None:
         self._led.value = value
 
     @staticmethod
-    def _sine_sample(length):
+    def _sine_sample(length) -> None:
         tone_volume = (2**15) - 1
         # Amplitude shift up in order to not have negative numbers
         shift = 2**15
         for i in range(length):
             yield int(tone_volume * math.sin(2 * math.pi * (i / length)) + shift)
 
     @staticmethod
-    def _square_sample(length):
+    def _square_sample(length) -> None:
```

Comment:
Same thing here about length and the return type:

```suggestion
    def _square_sample(length: int) -> Iterator[int]:
```